LeaderboardBackend/Services/Impl/UserService.cs

```diff
@@ -23,6 +24,30 @@ public UserService(ApplicationContext applicationContext, IAuthService authServi
         return await _applicationContext.Users.FindAsync(id);
     }
 
+    public async Task<GetUserResult> GetRegisteredUserFromClaims(ClaimsPrincipal claims)
```

Comment:
Debating whether we should simply have a GetUserFromClaims method and then just check the returned user's role. That would allow us to avoid duplicating logic.

Reply:
Yeah; as I've replied in https://github.com/leaderboardsgg/leaderboard-backend/pull/172#discussion_r1296998557, it doesn't really make sense for this method to exist. We can just get the user from claims, and then only verify their role in the create confirmation + send email method.

Reply:
I've renamed it to `GetUserFromClaims`